lib/moon/design/tabs/list.ex

```diff
@@ -30,6 +31,7 @@ defmodule Moon.Design.Tabs.List do
       {=@id}
       data-testid={@testid}
     >
+      <#slot {@default} />
```

Comment:
please put size to context here and also move default slot after other displaying methods